webroot/adm/oncall/generate-api-key-secret.html

```diff
@@ -0,0 +1,661 @@
+                                <li>If a site exists use the Site Id to update the site if necessary.</li>
+                                <li>Use the Site Id to create a client API key and secret if necessary.</li>
+                            </ol>
+                        </div>
+                    </div>
+                    <div class="row py-1">
+                        <div class="col">
+                            <h5>Search for existing participants:</h5>
+                        </div>
+                    </div>
+                    <div class="row align-items-end">
+                        <div class="col">
+                            <div class="form-group">
+                                <label for="participantName">Participant / Client name</label>
+                                <input type="text" class="form-control" id="participantName"
+                                    placeholder="Leave empty for all participants / clients">
+                            </div>
+                        </div>
+                        <div class="col">
+                            <a href="#" class="btn btn-primary" id="searchParticipant">Search</a>
+                        </div>
+                    </div>
+                    <div class="row pt-3">
+                        <div class="col">
+                            <h5>Search Results:</h5>
+                        </div>
+                    </div>
+                    <div class="row pt-3 px-2 collapse" id="searchSiteNoSiteFoundAlert">
+                        <div class="alert alert-warning d-flex align-items-center pt-3" role="alert">
+                            <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Warning:">
+                                <use xlink:href="#exclamation-triangle-fill" />
+                            </svg>
+                            <div>
+                                No site found
+                            </div>
+                        </div>
+                    </div>
+                    <div class="row pt-3 px-2 collapse" id="searchSiteErrorAlert">
+                        <div class="alert alert-danger d-flex align-items-center pt-3" role="alert">
+                            <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Warning:">
+                                <use xlink:href="#exclamation-triangle-fill" />
+                            </svg>
+                            <div id="searchOutputErrorDiv">
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="row px-2 table-responsive">
+                        <table class="table table-bordered" id="searchResultsTable">
+                            <thead>
+                                <th scope="col" class="col-1">Sites</th>
+                                <th scope="col" class="col-1">Clients</th>
```

Comment:
I think this should be "Client Keys" or "API Keys"

Reply:
In the Portal and our external docs, we call them "API Keys". I think consistency would be good